Tests.Unit.Core.ApplicationServices/ApplicationServices/Contract/ItContractWriteServiceTest.cs

```diff
@@ -235,6 +249,284 @@ public void Cannot_Create_With_Parent_If_Different_Organizations()
             AssertFailureWithKnownErrorDetails(result, "Failed to set parent with Uuid:", OperationFailure.BadInput, transaction);
         }
 
+        [Theory]
+        [InlineData(true, true, true, true, true)]
+        [InlineData(true, true, true, true, false)]
+        [InlineData(true, true, true, false, true)]
+        [InlineData(true, true, false, true, true)]
+        [InlineData(true, false, true, true, true)]
+        [InlineData(false, true, true, true, true)]
+        [InlineData(false, false, false, false, false)]
+        public void Can_Create_With_GeneralData(bool withContractType, bool withContractTemplate, bool withAgreementElements, bool withValidFrom, bool withValidTo)
+        {
+            // Arrange
+            var (organizationUuid, itContractModificationParameters, createdContract, transaction) = SetupCreateScenarioPrerequisites();
+
+            var contractId = A<string>();
```

Comment:
Der kan ikke laves en hjælpemetode til dette? Vil gøre det noget lettere når vi skal lave de to "store" tests